helmfile/charts/init-harbor/templates/init-harbor-job.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: {{ .Release.Name }}-job
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
I can't quite remember what the init job does but have you checked that it handles being run multiple times correctly?